lib-ui-compose/src/commonMain/kotlin/com/ustadmobile/libuicompose/view/report/reportedit/ReportEditScreen.kt

```diff
@@ -0,0 +1,331 @@
+                            val updatedSeries = seriesItem.copy(reportTimeRange = selectedTimeRange)
+                            onSeriesChanged(updatedSeries)
+                        }
+                    )
+                }
+            }
+
+            // Filters Section
+            item {
+                if (!seriesItem.reportSeriesFilters.isNullOrEmpty()) {
+                    Text(
+                        text = stringResource(MR.strings.filters),
+                        modifier = Modifier.defaultScreenPadding()
+                    )
+                }
+                seriesItem.reportSeriesFilters?.forEachIndexed { index, reportFilter2 ->
+                    Row(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .defaultScreenPadding()
+
+                    ) {
+                        val fieldName = reportFilter2.reportFilterField?.name?.lowercase()
+                            ?.replaceFirstChar { it.uppercase() } ?: ""
+                        val comparisonSymbol =
+                            reportFilter2.reportFilterCondition?.let { getComparisonSymbol(it) }
+                        val filterText =
+                            "$fieldName $comparisonSymbol ${reportFilter2.reportFilterValue}"
+
+                        Text(
+                            text = filterText,
+                            modifier = Modifier.weight(1f)
+                        )
+                        Icon(
+                            imageVector = Icons.Filled.Close,
+                            contentDescription = "Remove filter",
+                            modifier = Modifier
+                                .clickable {
+                                    onRemoveFilter(index, seriesItem.reportSeriesUid)
+                                }
+                        )
+                    }
+                }
+            }
+
+            item {
+                Button(
+                    onClick = { onAddFilter(seriesItem.reportSeriesUid) },
+                    modifier = Modifier.fillMaxWidth()
+                ) {
+                    Text(
+                        text = stringResource(MR.strings.add_filter),
+                    )
+                }
+            }
+        }
+
+        item {
+            Button(onClick = { onAddSeries() }, modifier = Modifier.fillMaxWidth()) {
+                Text(
+                    text = stringResource(MR.strings.add_series),
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun EditReportDropdown(
+    value: Int,
+    label: String,
+    options: List<MessageIdOption2>,
+    onOptionSelected: (MessageIdOption2) -> Unit,
+    modifier: Modifier = Modifier,
+    isError: Boolean = false,
+    enabled: Boolean = true,
+    supportingText: (@Composable () -> Unit)? = null,
+) {
+    Column(modifier = Modifier) {
+        Text(label, fontWeight = FontWeight.SemiBold)
```

Comment:
As I have mentioned previously: always use the theme and **never** set styles manually unless it is explicitly noted otherwise in the task card or prototype. We must **always** use the theme to ensure that dark mode, light mode, user font size settings, etc are handled properly and manually setting fonts / styles at best will make the application look inconsistent, at worse will start breaking things.